pyinjective/ofac.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+import json
+import os
+
+import aiohttp
+
+
+class OfacChecker:
+    OFAC_LIST_URL = "https://raw.githubusercontent.com/InjectiveLabs/injective-lists/master/wallets/ofac.json"
+    OFAC_LIST_FILENAME_DEFAULT = "ofac.json"
+    OFAC_LIST_FILENAME = OFAC_LIST_FILENAME_DEFAULT
+
+    def __init__(self):
+        self._ofac_list_path = self.get_ofac_list_path()
+        if not os.path.exists(self._ofac_list_path):
+            raise Exception(
+                "OFAC list is missing on the disk. Please, download it by running python3 pyinjective/ofac_list.py"
+            )
+
+        with open(self._ofac_list_path, "r") as f:
+            self._ofac_list = set(json.load(f))
+
+    @classmethod
+    def set_up(cls, ofac_list_filename: str):
+        OfacChecker.OFAC_LIST_FILENAME = ofac_list_filename
+
+    @classmethod
+    def tear_down(cls):
+        OfacChecker.OFAC_LIST_FILENAME = OfacChecker.OFAC_LIST_FILENAME_DEFAULT
```

Comment:
I am not sure I understand the reason to have "set up" and "tear down" methods here. Those methods are only expected in test classes.

Reply:
removed these from the class